### ISSUE TITLE
upgrade to pyo3 0.9?

### ISSUE THREAD

**Issue by awaited-hare** [gperinazzo/dict-derive#10]
`pyo3` 0.9 is out. Any plan to update the dependency?

**Comment by gperinazzo** [comment]
The crate doesn't depend on PyO3 directly, only for tests. It should use the version of PyO3 that is on your project.

Are you having any issues with the derives when using PyO3 0.9?

**Comment by awaited-hare** [comment]
I thought 0.9 is not supported when I saw the pyo3 version in the project README is 0.8 before I actually start adding this crate into my project.

Thanks for your prompt reply and feel free to close the issue :)

**Comment by gperinazzo** [comment]
Glad I could help! I'll update the version on tests and README. Feel free to open another issue if you have any problems.
